cli/src/commands/commit.rs

```diff
@@ -28,7 +28,7 @@ use crate::ui::Ui;
 
 /// Update the description and create a new change on top.
 #[derive(clap::Args, Clone, Debug)]
-#[command(visible_aliases=&["ci"])]
+#[command(alias = "ci")]
```

Comment:
nit: maybe we don't have to hide `ci` because it doesn't conflict with any other commands?

Reply:
True. I feel like we should still hide it, move it to config, or remove it completely as we've done with many other aliases. But that would be for different reasons, so should be a separate PR.